file_paths.md

```diff
@@ -163,156 +163,166 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
-        inc *                                 # facter headers
-            facter *
-        lib *
-            facter.rb *                       # ruby bindings
-
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
 
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
-
-    C:\Program Files\Puppet Labs\Puppet\misc *
-        LICENSE.rtf *                         # license
-        puppetlabs.ico *                      # icon for start menu shortcut
-        puppetres.dll *                       # event log message resource dll
-        versions.txt *                        # versions of components
+    C:\Program Files\Puppet Labs\puppet *     # ruby-puppet root
+        bin *                                 # executables and dlls
+          c_rehash *
+          catstomp *
+          catstomp.bat *
+          curl.exe *
+          curl-config *
+          environment.bat *                   # setup LOAD_PATH, globally used by all our .bat files, to be symlinked to C:\Program Files\Puppet Labs\bin
+          erb.bat *
+          extlookup2hiera *
+          extlookup2hiera.bat *
+          facter.exe *                        # Facter executable (we need to add facter.bat also I think)
+          gem.bat *
+          hiera *                             # ruby bin wrapper
+          hiera.bat *                         # To be symlinked to C:\Program Files\Puppet Labs\bin
+          irb.bat *
+          libcpp-pcp-client.so *
+          libcurl-4.dll *                     # curl dll
+          libeay32.dll *                      # openssl dll
+          libfacter.so *
+          mco *                               # ruby bin wrapper
+          mco.bat *                           # To be symlinked to C:\Program Files\Puppet Labs\bin
+          mcollectived *                      # ruby bin wrapper
+          minitar *
+          minitar.bat *
+          openssl.exe *
+          puppet *                            # ruby bin wrapper
+          puppet.bat *                        # To be symlinked to C:\Program Files\Puppet Labs\bin
+          puppetres.dll *                     # event log message resource dll
+          pxp-agent.exe *
+          rake.bat *
+          rdoc.bat *
+          ri.bat *
+          ruby.exe *
+          rubyw.exe *
+          ssleay32.dll *                      # openssl dll
+          stompcat *
+          stompcat.bat *
+          testrb.bat *
+          x64-msvcrt-ruby210.dll *            # ruby dll
+          *.dll *                             # dependency libraries (zlib, gcc, mingw, boost, etc.)
+
+        cache *                               # :vardir                      C:\ProgramData\PuppetLabs\puppet\var
```

Comment:
This was moved from `C:\ProgramData\PuppetLabs\puppet\cache` to `C:\Program Files\Puppet Labs\puppet\cache`